### PR TITLE
Link to v1.pantsbuild.org docs

### LINKED ISSUE #10: v1 docs have disappeared
From #4:

1.30 is not marked as deprecated on the current docs, but doesn't appear to be preserved and redirects don't work. Are we just giving up on 1.30 entirely? e.g. https://www.pantsbuild.org/v1.30/docs -> https://docs.pantsbuild.org/v1.30/docs

### PR DESCRIPTION
**Pull request by huonw**
This fixes #10 by just linking to the https://v1.pantsbuild.org archive from the all-versions page, as a good-enough solution for supporting people still on v1.

It doesn't attempt to redirect the links from the README v1 site, e.g. https://www.pantsbuild.org/v1.30/docs/welcome-to-pants. Per https://github.com/pantsbuild/pantsbuild.org/issues/10#issuecomment-1887787546, there's not many page views.

### CODE REVIEW

**Comment by thejcannon** [comment]
I'm +1.

However, instead of marking this as fixing the issue, I think we can save that for another PR (which I can open tonight) where we swizzle `NotFound/Content` and add some helpful info (e.g. "looking for X try Y?")

File: src/pages/versions/index.md [hunk 1]
```diff
@@ -20,3 +20,4 @@
...
 | 2.0           | [/2.0](/2.0.x/docs/introduction/how-does-pants-work)   | [2.0.x.md](https://github.com/pantsbuild/pants/blob/main/src/python/pants/notes/2.0.x.rst)  | [How to upgrade from Pants 1 to Pants 2](https://www.pantsbuild.org/v1.30/docs/how-to-upgrade-to-the-v2-engine)<br/>[Blog: Introducing Pants v2](https://blog.pantsbuild.org/introducing-pants-v2/)                                                                                                                                                |
+| 1.30          | [archive](https://v1.pantsbuild.org)                   |                                                                                             |                                                                                                                                                                                                                                                                                                                                                    |
```
Comment (thejcannon) [inline src/pages/versions/index.md:23]: Want to add 1.27-1.29 as well? Or just mark this `1.x`?

**Review by thejcannon — APPROVED** [review]

File: src/pages/versions/index.md (hunk 1 above)
Comment (huonw) [reply src/pages/versions/index.md:23]: Are you suggesting one of these?

```suggestion
| 1.0 - 1.30    | [archive](https://v1.pantsbuild.org)                   |                                                                                             |                                                                                                                                                                                                                                                                                                                                                    |
```

```suggestion
| 1.x           | [archive](https://v1.pantsbuild.org)                   |                                                                                             |                                                                                                                                                                                                                                                                                                                                                    |
```

Or maybe:

```suggestion
| ≤ 1.30        | [archive](https://v1.pantsbuild.org)                   |                                                                                             |                                                                                                                                                                                                                                                                                                                                                    |
```

Comment (thejcannon) [reply src/pages/versions/index.md:23]: Any of those would suffice for me.

Dealers choice

**Comment by huonw** [comment]
> However, instead of marking this as fixing the issue, I think we can save that for another PR (which I can open tonight) where we swizzle NotFound/Content and add some helpful info (e.g. "looking for X try Y?")

I've filed https://github.com/pantsbuild/pantsbuild.org/issues/88 to cover this, since it seems independently useful, not just for v1.

As such, I'll leave this one as closing #10, as it feels like good-enough for shipping the new site.

**Pull request merged**